tests/block_log_retain_blocks_test.py

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env python3
+
+from testUtils import Utils
+from Cluster import Cluster
+from WalletMgr import WalletMgr
+from TestHelper import TestHelper
+
+import random
+import os
+import signal
+
+###############################################################
+# terminate-scenarios-test
+#
+# A basic test for --block-log-retain-blocks option. It validates
+#   * no blocks.log is generated when the option is set to 0
+#   * blocks.log is generated when the option is set to greater than 0
+#   * blocks.log is generated when the option is not present.
+#
+###############################################################
+
+Print=Utils.Print
+errorExit=Utils.errorExit
+
+args=TestHelper.parse_args({"--keep-logs" ,"--dump-error-details","-v","--leave-running","--clean-run" })
+debug=args.v
+killEosInstances= not args.leave_running
+dumpErrorDetails=args.dump_error_details
+keepLogs=args.keep_logs
+killAll=args.clean_run
+
+seed=1
+Utils.Debug=debug
+testSuccessful=False
+
+random.seed(seed) # Use a fixed seed for repeatability.
+cluster=Cluster(walletd=True)
+walletMgr=WalletMgr(True)
+
+# the first  node for --block-log-retain-blocks 0,
+# the second for --block-log-retain-blocks 10,
+# the third for -block-log-retain-blocks not configured
+pnodes=1
+total_nodes=pnodes + 2
+
+try:
+    TestHelper.printSystemInfo("BEGIN")
+    cluster.setWalletMgr(walletMgr)
+
+    cluster.setWalletMgr(walletMgr)
```

Comment:
Duplicate line.

Reply:
Thanks for careful reviewing! I am fixing it.